standard_product_localizer.py

```diff
@@ -49,7 +49,7 @@ def filter(self, record):
 class ACQ:
     def __init__(self, acq_id, acq_type):
         self.acq_id=acq_id
-	self.acq_type = acq_type
+        self.acq_type = acq_type
 
 def get_acq_object(acq_id, acq_type):
```

Comment:
To resolve this issue when running localizer on Python 3.7.6 on latest hysds isce2 image:
```
Traceback (most recent call last):
  File "initiate_standard_product_localizer.py", line 4, in <module>
    import standard_product_localizer
  File "/home/ops/verdi/ops/standard_product/standard_product_localizer.py", line 52
    self.acq_type = acq_type
                           ^
TabError: inconsistent use of tabs and spaces in indentation
```